services/worker-manager/schemas/v1/worker-pool-full.yml

```diff
@@ -62,6 +62,46 @@ properties:
     type: integer
     minimum: 0
     description: Total capacity available across all workers for this worker pool that are currently not "stopped"
+  requestedCount:
+    title: Requested Worker Count
+    type: integer
+    minimum: 0
+    description: Total woker count in "requested" state for this worker pool
```

Comment:
s/woker/worker/

Are the various worker states defined somewhere that we can link to? I think this could be useful context for the docs, or a few words about e.g. what the requested state represents. I don't know if we already have such a document, but ideally we'd have a reference doc with a finite state diagram showing how the workers transition through the states, together with a description of what each state means, and then we'd reference that in these descriptions, so the reader knows what the different states mean. If we don't already have something like that, I think just having a few words about what the state means would also have value, as I realise writing a new doc is a bit of scope creep, and we could do that in a different (follow-up) PR.